pages/configuration/multinode_setup.rst

```diff
@@ -57,9 +57,17 @@ When you secure your Elasticsearch with `User Authentication <https://www.elasti
 Graylog Multi-node
 ==================
 
-After the installation of Graylog, you should take care that only one Graylog node is configured to be master with the configuration setting ``is_master = true``.
+After the installation of Graylog, you should take care that only one Graylog node is configured to be master with the
+configuration setting ``is_master = true``.
 
-The HTTP settings configured in ``http_bind_address`` (or ``http_publish_uri``) must be accessable for all Graylog nodes of the cluster.
+The ``http_bind_address`` configured address needs to be reachable by all Graylog nodes in the cluster.
+Same for the ``http_publish_uri`` which is mostly the the ``http_bind_address`` prefixed with the protocol and a closing
```

Comment:
The the
```suggestion
Same for the ``http_publish_uri`` which is mostly the ``http_bind_address`` prefixed with the protocol and a closing
```

But how about:
The ``http_publish_uri`` is normally auto-generated from the ``http_bind_address``.
This URI is used for the inter-node communication.